src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketScmFormValidationDelegate.java

```diff
@@ -52,7 +52,7 @@ public FormValidation doCheckCredentialsId(String credentialsId) {
     public FormValidation doCheckProjectName(String serverId, String credentialsId, String projectName) {
         Jenkins.get().checkPermission(CONFIGURE);
         if (isBlank(projectName)) {
-            return FormValidation.error("Please specify a project name.");
+            return FormValidation.error("Required");
```

Comment:
Same reasoning as before. Because the user sees the label, help text, and validation all at once I think it is better to simplify here and to leave this as is.